src/pytest_elasticsearch/factories.py

```diff
@@ -190,3 +190,32 @@ def drop_indexes():
         return client
 
     return elasticsearch_fixture
+
+
+def async_elasticsearch(process_fixture_name):
```

Comment:
I'm wondering what would be the use case for the async fixture? To use the async connection for the code that expects it?

Reply:
Yes, exactly . So recently, changed alot of our ES interface function to utilise async. So needed this fixture to be able to actually test this new functions.